markets/perps-market/contracts/storage/LiquidationAssetManager.sol

```diff
@@ -53,9 +53,6 @@ library LiquidationAssetManager {
             revert ParameterError.InvalidParameter("collateralTypes", "must not be empty");
         }
         for (uint256 i = 0; i < collateralTypesLength; ) {
-            if (self.poolDelegatedCollateralTypes[i] == address(0)) {
-                revert AddressError.ZeroAddress();
```

Comment:
later in the code there is an iteration over all the collaterals configured and then their prices are used. So we end up having error that Zero-address oracle node is used.

Reply:
so to use pool level distributors with perps we would need quite a bit more refactoring